### ISSUE TITLE
KeyError: 'token' when accessing webhooks

### ISSUE THREAD

**Issue by Heath123** [Rapptz/discord.py#2364]
### Summary
I get KeyError: 'token'

### Reproduction Steps

```    categorylist=[]
    channelslist=[]
    categories={}
    try:
      guildhooks = await guild.webhooks() # ERROR HAPPENS HERE
    except Exception as e:
      traceback.print_exc(file=sys.stderr)
      guildhooks = []
    for chnl in guild.channels:
      if chnl.id != 421771267100901379:
        continue
```

### Expected Results

It should work and return the webhooks

### Actual Results

```Traceback (most recent call last):
  File "/home/user/discross/discordmodule/__init__.py", line 326, in updateguild
    guildhooks = await guild.webhooks()
  File "/usr/local/lib/python3.7/site-packages/discord/guild.py", line 1317, in webhooks
    return [Webhook.from_state(d, state=self._state) for d in data]
  File "/usr/local/lib/python3.7/site-packages/discord/guild.py", line 1317, in <listcomp>
    return [Webhook.from_state(d, state=self._state) for d in data]
  File "/usr/local/lib/python3.7/site-packages/discord/webhook.py", line 500, in from_state
    return cls(data, adapter=AsyncWebhookAdapter(session=session), state=state)
  File "/usr/local/lib/python3.7/site-packages/discord/webhook.py", line 424, in __init__
    self.token = data['token']
KeyError: 'token'
```

### Checklist

- [x] I have searched the open issues for duplicates.
- [x] I have shown the entire traceback, if possible.
- [x] I have removed my token from display, if visible.

### System Information

```- Python v3.7.4-final
- discord.py v1.2.3-final
- aiohttp v3.5.4
- websockets v8.0.2
- system info: Linux 4.9.0-11-amd64 #1 SMP Debian 4.9.189-3+deb9u1 (2019-09-20)
```

**Comment by Phxntxm** [comment]
This is something that was fixed in version 1.3, either wait till it's released or you can install the dev version via this command:
```
pip install -U git+https://github.com/Rapptz/discord.py@master#egg=discord.py[voice]
```

**Comment by Heath123** [comment]
That works. Thank you!
